src/sinks/util/service.rs

```diff
@@ -174,39 +71,72 @@ pub struct TowerRequestConfig<T: ConcurrencyOption = Concurrency> {
     #[serde(skip_serializing_if = "ConcurrencyOption::is_none")]
     pub in_flight_limit: T, // 5
     pub timeout_secs: Option<u64>,             // 60
-    pub rate_limit_duration_secs: Option<u64>, // 1
-    pub rate_limit_num: Option<u64>,           // 5
-    pub retry_attempts: Option<usize>,         // max_value()
+    pub rate_limit_duration_secs: Option<u64>, // 1 day
+    pub rate_limit_num: Option<u64>,           // u64::MAX
+    pub retry_attempts: Option<usize>,         // usize::MAX
```

Comment:
Pendant nit: isn't this really `isize::MAX` given the definition of `RETRY_ATTEMPTS_DEFAULT` below?

Reply:
Ah yes, and `i64`. I will adjust.